src/screens/Home.tsx

```diff
@@ -24,4 +38,25 @@ const Home = (): JSX.Element => {
     </View>
   );
 };
+
+//not sure where I should put this will leave here for now 
```

Comment:
Great React code! We can leave this here for now and we can add the font assets in the next PR